From linux-kernel-owner@vger.kernel.org Fri Jul 14 09:31:36 2006
Date: Sat, 15 Jul 2006 01:30:11 +0900
From: Yoichi Yuasa <yoichi_yuasa@tripeaks.co.jp>
To: Lexington Luthor <Lexington.Luthor@gmail.com>
Subject: Driver core: fixed add_bind_files() definition
Message-Id: <20060715013011.044be9ad.yoichi_yuasa@tripeaks.co.jp>

When CONFIG_HOTPLUG is n, add_bind_files() definition is wrong.
This patch has fixed it.

Signed-off-by: Yoichi Yuasa <yoichi_yuasa@tripeaks.co.jp>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/bus.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/base/bus.c
+++ gregkh-2.6/drivers/base/bus.c
@@ -467,7 +467,7 @@ static void remove_bind_files(struct dev
 	driver_remove_file(drv, &driver_attr_unbind);
 }
 #else
-static inline void add_bind_files(struct device_driver *drv) {}
+static inline int add_bind_files(struct device_driver *drv) { return 0; }
 static inline void remove_bind_files(struct device_driver *drv) {}
 #endif
 
